app/components/SearchInput/Input.tsx

```diff
@@ -19,14 +19,16 @@ interface SearchInputProps {
 }
 export const SearchInput = ({onChange, expandable, placeholderText}: SearchInputProps) => {
   const [search, setSearch] = useState('')
-
+  const inputRef = useRef<HTMLInputElement | null>(null)
```

Comment:
try `const inputRef = useRef<HTMLInputElement>()` - undefined is better for this kind of thing

Reply:
that's a lint error